docs/enhancement-proposals/off-ep-0000.md

```diff
@@ -0,0 +1,92 @@
+  ``System`` models.
+* the SMIRNOFF force field specification.
+* the standard operating procedure for curating and adopting data sets, such as the 
+  [QCA Standards](https://github.com/openforcefield/qca-dataset-submission/blob/master/STANDARDS.md).
+  
+Each OFF-EP should clearly and concisely outline what new standard is being introduced
+or updated, and most importantly, provide a rationale for the change and get input
+from relevant stakeholders.
+
+Because the OFF-EPs are maintained as text files in a versioned repository,
+their revision history is the historical record of the standards proposal.
+
+### Types
+
+There are three kinds of OFF-EPs:
+
+1. **Standards Track** OFF-EPs describe a new feature or implementation
+1. **Informational** OFF-EPs provide general, non-binding guidance to the community but do not propose a new feature.
+1. **Process** OFF-EPs describe a change that may not be technical in nature but requires community input.
+
+[comment]: <> (The types are copied from PEPs, which are more elaborate. Can we just drop this section? i.e. is it worth the overhead to separate out EPs by these distinctions?)
+
+
+## Workflow
+
+The OFF-EP process begins with a new idea for Open Force Field infrastructure and/or community.
+It is highly recommended that a single NEP contain a single key proposal or new idea with a concise scope.
+Small changes often do not need to go through the OFF-EP process and can be submitted directly to as either a question to the community or a software patch.
+Each OFF-EP must have a champion -- someone who writes the proposal using the template provded below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the proposal.
+
+### Submission
+
+Once a proposal is written, this draft should be submitted as a pull request to the [openforcefield/standards](https://github.com/openforcefield/standards) repository with the status set to `Proposed`.
+It shall be merged at the earliest convenience of maintainers (note that this merge does not imply acceptance, as its status shall still be `Proposed`.)
+At this point, members of the Open Force Field community will review the submission.
+
+### Review
+
+### Resolution
+
+All OFF-EPs will be resolved as either *Rejected*, *Accepted* or *Deferred*
+
+An OFF-EP is accepted upon consensus approval of the stakeholders.
+The authors of a NEP are responsible for defining a consensus threshold.
+
+[comment]: <> (Potentially tricky; PEP and NEP do not make acceptance guidelines clear)
+
+In some cases unanimity may be necessary, in others some strong majority may be sufficient.
+A stakeholder(s) holding genuine reservations about any portion of the EP may block its acceptance, overriding the acceptance criteria laid out by the author(s).
+
+In unusual cases, OpenFF PIs may be asked to decide whether a controversial NEP is `Accepted` and may elect to resolve the status to `Accepted` or `Deferred`.
```

Comment:
This is complicated. Let's keep it simple for now.

```suggestion
An OFF-EP is accepted upon unanimous approval of @SimonBoothroyd @j-wags @davidlmobley and @jchodera. 
```

Reply:
This seems reasonable, though what if we couldn't agree? That scenario may not need formalization though.

Reply:
Perhaps add "This acceptance criterion is expected to be updated/amended as the community grows and changes."